From 6cdd7ee76dc39b95887d9c564975089fba0fe213 Mon Sep 17 00:00:00 2001
From: "Ruan, Hongfu" <hongfu.ruan@intel.com>
Date: Thu, 2 Dec 2021 10:44:50 +0800
Subject: [PATCH] update policy for 3rd party App function

this change works together with changes under system/sepolicy, or
it doesn't pass compilation.

Tracked-On: OAM-100126
Signed-off-by: Ruan, Hongfu <hongfu.ruan@intel.com>
---
 houdini/vendor_init.te | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/houdini/vendor_init.te b/houdini/vendor_init.te
index 66d5c83..71f569a 100644
--- a/houdini/vendor_init.te
+++ b/houdini/vendor_init.te
@@ -3,3 +3,15 @@ set_prop(vendor_init, exported_dalvik_prop)
 
 allow vendor_init binfmt_miscfs:file write;
 allow vendor_init binfmt_miscfs:chr_file { create setattr unlink rw_file_perms };
+
+allow untrusted_app_27 sysfs_net:dir search;
+allow untrusted_app_27 sysfs_rtc:dir search;
+allow untrusted_app_27 sysfs_rtc:file read;
+allow untrusted_app_27 sysfs_rtc:file open;
+allow untrusted_app_27 sysfs_rtc:file getattr;
+allow untrusted_app_27 vendor_file:file execute;
+allow untrusted_app_27 su_exec:file getattr;
+allow untrusted_app_27 sysfs:dir read;
+allow untrusted_app_27 vendor_file:file { map open read };
+allow untrusted_app_27 vendor_file:file getattr;
+
-- 
2.29.2

